doc/config_settings.xml

```diff
@@ -952,6 +952,18 @@
         <listitem>Border stippling (dashing) in pixels
         <para /></listitem>
     </varlistentry>
+    <varlistentry>
+        <term>
+            <command>
+                <option>render_draw_blended</option>
```

Comment:
(nit) `r` goes before `s`.

Reply:
(question) Would `blend_images` make a better option name?

Reply:
Updated the ordering, whoops.

`draw_blended` or some variation seems OK to me. At first I thought there was another blend going on that it should be separate from, but that's really just part of loading the image from the file to the buffer it seems, so maybe it doesn't need to be specific to `render`.